src/components/film/film.service.ts

```diff
@@ -91,7 +63,7 @@ export class FilmService {
   }
 
   async update(input: UpdateFilm, session: Session): Promise<Film> {
-    const film = await this.readOne(input.id, session);
+    const film = await this.repo.readOne(input.id);
```

Comment:
@CarsonF - Any advice on how to proceed with the error below.  I think the issue is the start/end vs range issue?

Reply:
Yeah it's because `secure()` also parses the scripture range IDs back into objects.
https://github.com/SeedCompany/cord-api-v3/pull/3185/files#diff-5578576cbb43b3c5ac0e42edf81d777514503b411c133ae9575f36a2906f2d82R59-R61